tests/conftest.py

```diff
@@ -54,30 +54,37 @@ def n_subsystems(request):
 def qubit_device(n_subsystems):
     return qml.device('default.qubit', wires=n_subsystems)
 
+
 @pytest.fixture(scope="function")
 def qubit_device_1_wire():
     return qml.device('default.qubit', wires=1)
 
+
 @pytest.fixture(scope="function")
 def qubit_device_2_wires():
     return qml.device('default.qubit', wires=2)
 
+
 @pytest.fixture(scope="function")
 def qubit_device_3_wires():
     return qml.device('default.qubit', wires=3)
 
+
 @pytest.fixture(scope="session")
 def tensornet_device(n_subsystems):
     return qml.device('expt.tensornet', wires=n_subsystems)
 
+
 @pytest.fixture(scope="function")
 def tensornet_device_1_wire():
     return qml.device('expt.tensornet', wires=1)
 
+
 @pytest.fixture(scope="function")
 def tensornet_device_2_wires():
     return qml.device('expt.tensornet', wires=2)
 
+
 @pytest.fixture(scope="function")
 def tensornet_device_3_wires():
     return qml.device('expt.tensornet', wires=3)
```

Comment:
Not relevant to this PR at all, but something that has been annoying me in general. I find these fixtures pointless, since they are simply wrapping a single line, and are executed on a per function basis. They also add noise to `conftest.py`, and add some developer overhead --- I'm constantly having to swap between test files and conftest.py.

I propose we delete these, and simply use `dev = qml.device(..)` in the integration tests directly.

Reply:
I am very much in favour. Except from the ``gaussian_device`` ones please, since they use a dummy device that has to be defined.

Reply:
Yep, dummy/mocked devices should remain fixtures :)